src/main/rx/cc2500_frsky_x.c

```diff
@@ -177,15 +177,11 @@ static void buildTelemetryFrame(uint8_t *packet)
     frame[0] = 0x0E;//length
     frame[1] = rxCc2500SpiConfig()->bindTxId[0];
     frame[2] = rxCc2500SpiConfig()->bindTxId[1];
-    switch(spiProtocol) {
-    case RX_SPI_FRSKY_X:
-    case RX_SPI_FRSKY_X_LBT:
+
+    if (spiProtocol == RX_SPI_FRSKY_X || spiProtocol == RX_SPI_FRSKY_X_LBT) {
         frame[3] = packet[3];
-        break;
-    case RX_SPI_FRSKY_X_V2:
-    case RX_SPI_FRSKY_X_LBT_V2:
+    } else if (spiProtocol == RX_SPI_FRSKY_X_V2 || spiProtocol == RX_SPI_FRSKY_X_LBT_V2) {
```

Comment:
The second if statemend can be removed. If the first will not be true it could be only the the other two for this part of the code.

Reply:
Agreed with @MJ666. Also, I'd turn them around and put it this way:

```
if (spiProtocol == RX_SPI_FRSKY_X_V2 || spiProtocol == RX_SPI_FRSKY_X_LBT_V2) {
    frame[3] = rxCc2500SpiConfig()->bindTxId[2];
} else {
    frame[3] = packet[3];
}
```

since `frame[3] = packet[3]` is the original default for the protocol.